tutorial/part1/chapter0_GitHub.md

```diff
@@ -31,11 +32,14 @@ Now that your file has been cloned to your computer, you need to initiate git by
 
 ![img-gitinit](./ch0_files/git-init.png)
 
-Do we need this section below?
+```{admonition} Editing TODO
```

Comment:
Git will ask for these credentials so we need this section.

Reply:
Agreed - we should leave this in. @Coding-with-Adam I think this was your suggestion right?

Reply:
I honestly can't remember if it was my suggestion or Shane's. But yes, we would need this section.